src/components/Comment/NestedComment/NestedComment.jsx

```diff
@@ -17,124 +14,89 @@ export default function NestedComment({
-              <Icon id='ellipsis-vertical' width='3' height='11' />
-            )}
-          </p>
-        </div>
-        <div className={styles.commentCenter}>
-          <div
-            className={`${styles.nestedPadding} ${(isDeleted || !isVisible) && styles.hide}`}
-          >
-            {!isDeleted &&
-              (isVisible ? content : '(관리자에 의해 차단된 댓글입니다)')}
-            {isDeleted && '(삭제된 댓글입니다)'}
+          <div className={styles.cloud}>
+            <Icon id='cloud' width='19' heigth='13' />
           </div>
+          <p className={`${isWriterWithdrawn && styles.isWriterWithdrawn}`}>
+            {isWriterWithdrawn ? '(알 수 없음)' : userDisplay}
+          </p>
+          <p className={styles.dot}>·</p>
+          <p>
+            {timeAgo(createdAt)} {isUpdated ? ' (수정됨)' : null}
+          </p>
         </div>
-        <div className={styles.commentBottom}>
-          {!isDeleted && (
-            <button
-              className={styles.likedCount}
-              onClick={() => (isLiked ? deleteLike.mutate() : like.mutate())}
-            >
-              <Icon
-                id='like'
-                width='13'
-                height='12'
-                fill={isLiked ? '#5F86BF' : '#D9D9D9'}
-              />
-              <span>{data.likeCount.toLocaleString()}</span>
-            </button>
+        <p
+          className={styles.dot3}
+          onClick={(e) => {
+            e.stopPropagation();
+            onCommentOptionClick(data);
+          }}
+        >
+          {!isDeleted && isVisible && (
+            <Icon id='ellipsis-vertical' width='3' height='11' />
           )}
+        </p>
+      </div>
+      <div className={styles.commentCenter}>
+        <div
+          className={`${styles.nestedPadding} ${(isDeleted || !isVisible) && styles.hide}`}
+        >
+          {!isDeleted &&
+            (isVisible ? content : '(관리자에 의해 차단된 댓글입니다)')}
+          {isDeleted && '(삭제된 댓글입니다)'}
         </div>
       </div>
-      <OptionModal
-        id='comment-edit'
-        isOpen={isOptionModalOpen}
-        setIsOpen={setIsOptionModalOpen}
-        closeFn={onCloseClick}
-        functions={{
-          pencil: () => {
-            setIsOptionModalOpen(false);
-            setIsEdit(true);
-            inputFocus();
-          },
-          trash: () => {
-            setIsDeleteModalOpen(true);
-            setIsOptionModalOpen(false);
-          },
-        }}
-      />
-      <DeleteModal
-        id='comment-delete'
-        isOpen={isDeleteModalOpen}
-        setIsOpen={setIsDeleteModalOpen}
-        redBtnFunction={() => {
-          deleteComment.mutate({ commentId });
-          setCommentId(undefined);
-          setContent('');
-        }}
-      />
-    </>
+      <div className={styles.commentBottom}>
+        {!isDeleted && (
+          <button
+            className={styles.likedCount}
+            onClick={() => (isLiked ? deleteLike.mutate() : like.mutate())}
+          >
```

Comment:
해당 button은 form안에서 `submit` 용도로 사용하고 있지 않기에, type을 `button`으로 작성하는 것이 더 명시적일 것 같습니다 😁
( button의 속성 중 type의 default value가 `submit` 입니다 )

참고글) https://nykim.work/96

```suggestion
          <button
           type="button" 
            className={styles.likedCount}
            onClick={() => (isLiked ? deleteLike.mutate() : like.mutate())}
          >
```